L1Trigger/Phase2L1GT/python/menuConstants.py

```diff
@@ -0,0 +1,103 @@
+import FWCore.ParameterSet.Config as cms
+
+from os import environ
+import json
+
+# print(os.path.exists("./scalings_v44.json"))
+scalings_fname = environ["CMSSW_BASE"] + "/src/L1Trigger/Phase2L1GT/python/scalings_v44.json"
+with open(scalings_fname, 'r') as file:
+    scalings = json.load(file)
```

Comment:
I don't think you are necessarily guaranteed this is always going to exist (with a partial checkout of CMSSW for example where you are not working with these directories directly). 

The JSON itself should be committed as a data file as it is not directly code.

I'm not sure if there is a FileInPath solution for inside a python configuration, but maybe one of the core people know.

Reply:
Should it be the `RELEASE_BASE` then instead? 

On the separation to `cms-data`: I think that given these constants are tied to the L1 menu, it would make sense to keep them together imo. Otherwise it might be hard to disentangle if changes are due to the menu config or constant changes.

Reply:
> I'm not sure if there is a FileInPath solution for inside a python configuration, but maybe one of the core people know.

there is `pfnInPath`.

 > Otherwise it might be hard to disentangle if changes are due to the menu config or constant changes.
 
I am not sure to see the issue. Can you elaborate? 
Having said that, the main draw-back of storing the configuration in `cms-data` is that a configuration change will require a full release as opposed to a patch-release. Having said that it's not clear to me what's the long term strategy for storing the phase-2 L1T menu, so it's hard to say if that is going to be an issue.

Reply:
@mmusich do you mean using this for example?
https://github.com/cms-sw/cmssw/blob/902185ebb53bc03dd80c6bfd5cb50a5667d572e3/Alignment/OfflineValidation/python/TkAlAllInOneTool/findAndChange.py#L55